packages/core/src/themeTokens.ts

```diff
@@ -0,0 +1,22 @@
+import { Theme } from "./theme";
+import { FlattenObject, NestedObject, Pretty } from "./utils/object";
+import { If, IsNever, _String, Stringify } from "./utils/types";
+import { Tokens } from "@kuma-ui/sheet";
+
+type ThemeTokens<T extends string> = Theme extends Record<T, unknown>
+  ? Theme[T]
+  : never;
+
+export type InputThemeTokens = Partial<Record<Tokens, NestedObject<string>>>;
```

Comment:
I thought tokens like `breakpoints` and `fontSize` should also accept `number` in theme.Depending on the property, it may be `string` or `string | number`.
Therefore, I thought it necessary to define the type of each token.